Autopilot/AttitudeManager/SendInstructionsToSafety.hpp

```diff
@@ -31,9 +32,10 @@ void SendToSafety_Init(void);
 * Communicates the desired actuator commands encoded in the channelOut array over I2C to
 * the safety chip. Internally, this function performs a conversion from percentages to pwm values.
 * This function is non blocking and returns right away.
-* @param[in]		channelOut 		the array (size of 4 floats) of percentages each channel should be set to.
+* @param[in]		channel 		the output channel for PWM
+* @param[in]		percent 		the percentage between 0-100 that the PWM channel should be set to
 * @return							the error struct, containing all info about any errors that may have occured.
 */
-SendToSafety_error_t SendToSafety_Execute(float *channelOut);
+SendToSafety_error_t SendToSafety_Execute(int channel, int percent);
```

Comment:
why make percent an integer ? It's not a big deal, but to me it looks like we're sacrificing some resolution for no upside.

Reply:
Interchip currently uses int's instead of floats since PWM manager uses int's. I could probably just make it take in a float and then round it if you'd like?

Reply:
understood, thanks. Keep it the way it is then!